src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Affix, Container, Flex, useMantineTheme } from '@mantine/core';
+import { Typography } from '../UI/Typography/Typography';
+import { Button } from '../UI/Button/Button';
+import { Divider } from '../UI/Divider/Divider';
+
+const quickLinks = [
+  { label: 'About', href: '/about' },
```

Comment:
The hrefs should come from the constant folder and the strings should come from the translation file, you can insert those keys into en json file 

Also extract the `quickLInks` object to another file under Footer folder